### DISCUSSION TITLE
cycle count still high

### DISCUSSION THREAD

**Discussion by yabo-boye** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#217] [Q&A]
I have been using this app for a few days trying to keep my battery between 30-70% (i have to use it unplugged often), but I am noticing the cycle count is still increasing in a way that is faster than I would have expected. Is there any tweaking I could do? 

I'm on a Mid-2012 Macbook Air, Macos Catalina, and a brand new battery.

**Comment by MatthiasKerbl** [comment]
Hi @yabo-boye,
I don't think there is more tweaking you could do. Using your battery between 30 and 70% sounds reasonable, maybe 20-60% is slightly better. Keep in mind, that the battery should endure up to 1000 cycles and keeping your SoC between these percentages will result in a slower degradation.

**Comment by danfoerster** [comment]
Unless you are having your Mac plugged in all day long, your battery is going to get cycles on it. To clarify what exactly one cycle is: Any amount of discharge and charge that add up to 100%.
Which means, if you drain your Mac 20% today, charge it again, use 20% tomorrow, use 40% the day after that, charge it again, and use 20% again and charge it, all that is going to add up to one cycle in a couple of days (This is how to keep charge cycles low). The adverse would be using 100% everyday and recharging once in the evening, which is going to result in one full cycle each day. This is the worst for your battery, as deep discharges (discharge below 20%) are bad and degrade the battery quickest. Disregarding any setting and preference you have set with AlDente, the best way to decrease the rate of getting charge cycles and keeping your battery heathy is:

- Plug it in whenever you can! You do not harm your battery by keeping your Mac plugged in. Its going to stop charging when its full anyways and draining your battery to empty is much worse than leaving it on a charger. Even much time on 100% isn't going to degrade the battery as quick as if you would drain it everyday.
- Quick top-off charges are better. Even if you can charge your Mac for only 15 minutes, do it.
- Make it habit of not draining the battery below 30% (unless you are doing a battery calibration).
- Make sure your Mac stays cool and the fan can blow air out of the machine unobstructed.
- Know that cycle count does not necessarily equal battery health. A device that has 300 cycles but has been charged to 100% and then always drained to 0 its entire life can be as bad or even worse than a battery that has not always been drained completely and which could already have 800 cycles.

For reference, I replaced my battery at the end of December. Granted the Mac has mostly been sitting at home but this is my battery stats according to coconutbattery. This is basically what you are aiming for. As of now, I gained about 5 charge cycles every month and lost about 0.6% battery health since January. (I have AlDente only since around the end of February. Only discharges were due to doing battery calibration and keeping the ions somewhat moving). And I noticed, the mAh always only drop significantly when the battery has been emptied completely. So I am pretty certain that draining it all the way is the worst thing you can do. Leaving it on 100% didn't do much harm in the first 2 months, or at least I didn't notice anything bad. But of course a steady percentage of maximum 80%, set via AlDente, is advised and that is why we are here :)

<img width="830" alt="Screen Shot 2021-03-21 at 20 21 31" src="https://user-images.githubusercontent.com/79795921/111918296-b7dd9380-8a84-11eb-9e5e-8bf636dfe5a6.png">

**Reply by vividfog** [reply]
I’ve had many Mac laptops and they lost about 10% a year when kept almost always plugged in at 100%. I can’t recommend that strategy. I got maybe 30-50 cycles a year. 

And that’s why AlDente is so awesome. Now the charging stops at 70% and the battery health is still at 100.3% after 4 months.

**Comment by yabo-boye** [comment]
wow everyone, thanks for the really in-depth and helpful explanations, i really appreciate it! 

does anyone know, what is the ideal battery temperature? i want to play with the macs fan control app to have an even more cool system (i live in a pretty hot tropical environment)

**Reply by danfoerster** [reply]
You're welcome
Before I try to explain it myself I'll just redirect you to this ;)
https://www.apple.com/batteries/maximizing-performance/
